### PR TITLE
Post bootstrap changes - remove headings from admin.scss

### LINKED ISSUE #19018: Remove headings from admin.scss
While working on removing bootstrap project, to make UI consistent we had to override headings h1-h6 in admin.scss . At that time it was decided that these headings will be removed in a later PR. [Reference](https://github.com/forem/forem/pull/18945#discussion_r1068293312)

## Task
- [ ] Remove h1-h6 from `app/assets/stylesheets/admin.scss`
- [ ] Make sure that all h1-h6 use consistent design guidelines

| Heading  | Crayons Style |
| ------------- | ------------- |
| h1 | crayons-title |
| h2 | crayons-subtitle-1 |
| h3 | crayons-subtitle-3 |
| h4 | crayons-subtitle-4 |

For h5 and h6 we don't have specific crayons styles right now but can be taken care of by writing custom css classes.

### PR DESCRIPTION
**Pull request by rt4914**
## What type of PR is this? (check all applicable)

- [x] Optimization

## Description

This PR removes all the headings that we are overriding in admin.scss file.

This PR uses following guidelines mainly (not fully).
| Heading  | Crayons Style |
| ------------- | ------------- |
| h1 | crayons-title |
| h2 | crayons-subtitle-1 |
| h3 | crayons-subtitle-3 |
| h4 | crayons-subtitle-4 |

## Related Tickets & Documents

- Closes #19018

## QA Instructions, Screenshots, Recordings

There are no major changes in UI but the code consistency and styles have been changed. 

**How can this best tested?**

- you can checkout in this branch, run the code locally as admin and check all admin pages/features and lookout for any weird UI.
- you can test it on https://canary1.forem.lol/admin/

### UI accessibility concerns?

Improves accessibility as there is exactly 1 h1 tag in all pages. And also the headings have a order of h1-h6, i.e. there won't be a page where we use h1 and then directly use h5 skipping h2,h3,h4 etc.

## Added/updated tests?

- [x] No

## [Forem core team only] How will this change be communicated?

- [x] I will share this change internally with the appropriate teams

## [optional] Are there any post deployment tasks we need to perform?

@maestromac We will need to deploy this on Canary1 or Canary2 to test.

### CODE REVIEW

**Comment by rt4914** [comment]
These changes have been approved by @anujbhavsar96 by doing QA on canary 1 [Reference](https://forem-team.slack.com/archives/C7GE84DEJ/p1675146136955909?thread_ts=1675085996.072309&cid=C7GE84DEJ)

File: app/views/admin/badge_achievements/index.html.erb [hunk 1]
```diff
@@ -38,7 +38,7 @@
         <td class="whitespace-nowrap"><%= badge_achievement.user.id %></td>
         <td><%= badge_achievement.user.username %></td>
         <td><%= badge_achievement.badge.title %></td>
-        <h5>
+        <div>
```
Comment (jaw6) [inline app/views/admin/badge_achievements/index.html.erb:41]: Probably minor, not a blocker: I don't think `div` is allowed inside a `tr` (except if inside a `td`), but I'm also not sure a `h5` is allowed inside a `tr` either, and I guess that was working. Maybe the question here is: what does this accomplish here?

**Review by jaw6 — APPROVED** [review]
One probably minor question, otherwise looks good to me :+1:

File: app/views/admin/badge_achievements/index.html.erb (hunk 1 above)
Comment (rt4914) [reply app/views/admin/badge_achievements/index.html.erb]: Nice catch @jaw6 . I did check and the `div` was actually getting ignored. Therefore I have removed it.
Screenshot 2023-02-02 at 3 27 01 PM

**Pull request merged**